server/index.js

```diff
@@ -17,13 +21,48 @@ module.exports = function Server(config, onStart) {
   // Enable CORS for all routes
   app.use(cors());
 
-  // Start server on conigured hose and port
-  const server = app.listen(config.port, config.host, function listen() {
-    this.rootUrl = `http://${this.address().address}:${this.address().port}`;
-    app.log('serve', `Listening at ${this.rootUrl}`);
-    // Execute callback once server starts
-    if (onStart) onStart.call(this);
-  });
+  // Start server on configured host and port
+  let server;
+
+  if (config.portHttps) {
+    let certFiles;
+    if (!config.httpsKeyPath && !config.httpsCertPath) {
+      certFiles = createCertFiles();
```

Comment:
@AndersDJohnson where are the cert files written? Project root?

Reply:
@ryanricard At `./node_modules/create-cert-files/*`.